std/json.d

```diff
@@ -57,6 +57,22 @@ import std.conv;
 import std.range.primitives;
 import std.array;
 import std.traits;
+import std.typecons : Flag;
+
+/**
+String literals used to represent special float values within JSON strings.
+*/
+enum JSONFloatLiteral : string
+{
+    nan         = "NaN",       /// string representation of floating-point NaN
+    inf         = "Infinite",  /// string representation of floating-point Infinity
+    negativeInf = "-Infinite", /// string representation of floating-point negative Infinity
+}
+
+/**
+Flag that enables encoding special float values (NaN/Inf) as strings.
+*/
+alias JsonSpecialFloats = Flag!"JsonSpecialFloats";
```

Comment:
The convention is `alias JsonSpecialFloats = Flag!"jsonSpecialFloats";`, not the small `j`, so that you can write `JsonSpecialFloats.yes` or `Yes.jsonSpecialFloats`.